corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -50,14 +50,42 @@ def html_name(name):
 
 
 @register.simple_tag
-def input_trans(name, langs=None, input_name='name'):
-    template = '''
-        <input type="text"
-               name="{}"
-               class="form-control"
-               value="%(value)s"
-               placeholder="%(placeholder)s" />
-    '''.format(input_name)
+def input_trans(name, langs=None, input_name='name', data_bind=None):
```

Comment:
If you think adding the `data_bind` parameter to `input_trans` is worth the complexity, maybe we should keep this commit, otherwise I'm happy just to drop it and rebase.